CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
-## UNRELEASED
-  - Releases from `main` are not desired until we get closer to Stack 9.x, and beginning the changelog with something other than release notes matching the version will prevent the plugin release tooling from allowing publication.
+## 9.0.0-prerelease
```

Comment:
(I am open to change naming)
@yaauie WDYT for:
- now we release prerelease versions
- between 2025-01-31 and 2025-02-5, beta1
- between 2025-02-5 and 2025-02-12, rc1
- after 2025-02-5, rc2 - this will not be embedded
- and finally 9.0.0 itself

For each release we need to embed to core and recreate BC.

Reply:
Or I think creating _prerelease_ and _beta1_ (between 2025-01-31 and 2025-02-5) seems sufficient (as plugin always goes its individual release cycle)